### DISCUSSION TITLE
Attacco tipo

### DISCUSSION THREAD

**Discussion by xrivo1** [rejetto/hfs#209] [General]
Per semplice conoscenza, allego log di tentativo di accesso. il tutto durato circa 90 secondi-
[access.log](https://github.com/rejetto/hfs/files/11185032/access.log)

**Comment by rejetto** [comment]
questo tipo di attacchi sono estremamente comuni, quanto innocui in quanto cercano (alla cieca) dei software specifici.
Se tu usi un dominio (di qualsiasi livello) puoi evitare di avere questo traffico indesiderato tramite il plugin vhosting.

**Comment by xrivo1** [comment]
Non uso un dominio. ho HFS installato su server casalingo e utilizzo DDNS per risolvere l'indirizzo IP esterno

**Comment by rejetto** [comment]
anche DDNS è un dominio.
Configurati vhosting con una semplice riga che per quel dominio, e poi abilita l'opzione "block requests...".
Solo chi conosce il tuo dominio verrà accettato.

**Comment by xrivo1** [comment]
Ottimo. Ti chiedo però cortesemente un aiuto in quanto non sono molto pratico di righe di comando. 
il DDNS è *******.homepc.it
come devo procedere?

**Reply by rejetto** [reply]
attiva il plugin,
vai nelle sue opzioni,
premi +
come domain metti il tuo dominio
come root metti /
salva
attiva "block..."
salva

**Reply by gogi100** [reply]
Can you explain what this plugin works? If external users access per ip address, he blocks them?

**Reply by rejetto** [reply]
exactly, but this is not the main purpose of the plugin.
The block is just an extra feature.

**Reply by xrivo1** [reply]
fantastico. funziona alla perfezione.

**Comment by gogi100** [comment]
Can you explain basic features of the plugin?

**Comment by rejetto** [comment]
the goal is to have different content for different domains
